rbac/management/principal/it_service.py

```diff
@@ -315,6 +317,24 @@ def get_service_accounts_group(self, group: Group, user: User, options: dict = {
         else:
             return service_accounts
 
+    def generate_service_accounts_report_in_group(self, group: Group, client_ids: set[UUID]) -> dict[str, bool]:
+        """Check if the given service accounts are in the specified group."""
+        # Fetch the service accounts from the group.
+        group_service_account_principals = (
+            group.principals.values_list("service_account_id", flat=True)
+            .filter(type=TYPE_SERVICE_ACCOUNT)
+            .filter(service_account_id__in=client_ids)
+        )
+
+        # Mark the specified client IDs as "present or missing" from the result set.ç
```

Comment:
`set.ç` in the comment is probably a typo?

Reply:
Yes, you're right. Good catch!